### PR TITLE
Added a new medication, Miner's Salve

### PR DESCRIPTION
**Pull request by lordpidey**
Adds a new medication, which will give doctors a new long term healing option, miners a way to make a medication on mining station, and will have some nefarious uses too for traitors.

### CODE REVIEW

File: code/modules/reagents/Chemistry-Recipes/Medicine.dm [hunk 1]
```diff
@@ -57,6 +57,22 @@
 	result = "salglu_solution"
 	required_reagents = list("sodiumchloride" = 1, "water" = 1, "sugar" = 1)
 	result_amount = 3
+	
+/datum/chemical_reaction/mine_salve
+	name = "Miner's Salve"
+	id = "mine_salve"
+	result = "mine_salve"
+	required_reagents = list("oil" = 1, "water" = 1, "iron" = 1)
+	mix_message = "The fine black particles dissolve into the oily substance."
```
Comment (ojacobson) [inline code/modules/reagents/Chemistry-Recipes/Medicine.dm]: Thematic niggle: these two messages (his, and the one in `mine_salve2`, below, are considerably more detailed than the other reaction messages. Keep in mind that these can be seen from several tiles away in some circumstances: does it make sense for someone at the chem door to be observing fine particles in anything?

I don't necessarily have an improved suggestion for you, but it stood out for me.

File: code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm [hunk 2]
```diff
@@ -223,6 +223,30 @@
...
+/datum/reagent/medicine/mine_salve
+	name = "Miner's Salve"
+	id = "mine_salve"
+	description = "Slowly heals burn and brute damage, and causes subject to believe they are fully healed."
+	reagent_state = LIQUID
+	color = "#6D6374"
+	metabolization_rate = 0.4 * REAGENTS_METABOLISM
+
+/datum/reagent/medicine/mine_salve/on_mob_life(mob/living/M)
+	if(istype(M, /mob/living/carbon))
```
Comment (Jordie0608) [inline code/modules/reagents/Chemistry-Reagents/Medicine-Reagents.dm]: There's an IsCarbon() check which does the same as this and is better to use for some reason.

File: code/modules/mob/living/carbon/human/species.dm [hunk 3]
```diff
@@ -737,6 +738,8 @@
 					icon_num = 4
 				if(damage > (comparison*4))
 					icon_num = 5
+				if(H.hal_screwyhud == 5)
```
Comment (ojacobson) [inline code/modules/mob/living/carbon/human/species.dm]: I like this. We have another med that gives misleading "you are healthy" indicators (Aranesp), and it's one of the more entertaining things to dose someone with.

**Comment by ojacobson** [comment]
I'm not sure how I feel about a recipe that suggests that oil (an easy-to-make early tier reagent) and plasma (notionally complicated, but in fact available from the chem vendor) are interchangeable. Given this recipe, I'll be curious if anyone keeps making saline-glucose mix.

**Comment by Chocobro** [comment]
how to do your first PR right

seriously this guy did his own changelog yml and actually bothered to ask in coderbus when he had trouble.

looks good to me. :+1:

**Comment by ojacobson** [comment]
> seriously this guy did his own changelog yml and actually bothered to ask in coderbus when he had trouble.

also this. :+1: :heavy_multiplication_x:  :100:

**Comment by WJohn** [comment]
Do we really need another tricord?

**Comment by lordpidey** [comment]
WJohn, it's a very slow and long lasting medication with unique side effects.  The side effects could easily be used by a traitor.

ojacobson, being able to use plasma for the recipe allows it to be made entirely on the mining station, hence it's name.  Throw a sheet of metal, plasma, and 20 units of water in a beaker and you've got some medication.

**Comment by phil235** [comment]
It's not exactly slow, its meabolization rate is between salglu and omnizine. And its healing power is lower than omnizine.

**Comment by Iamgoofball** [comment]
This isn't a tricord, WJ.

This is good. I like it. It has an upside and a downside. It can be used for traitoring or for healing.

:+1:

**Comment by SconesC** [comment]
Miners don't have a grinder though

**Comment by milocv** [comment]
Looks nice. I think there should be a lot more "ghetto chemistry" options.

**Comment by Jalleo** [comment]
This is probably one of the best medicines to hear about.

What are the side effects?

**Comment by Thunder12345** [comment]
It makes you think you're instantly at full health. Consider the effect of adding a little of that to a poison mix. They'll never even know they're dying...

**Comment by AlexanderUlanH** [comment]
'Salve' doesn't work as a touch application.  Might want to rename it.
Also, like Scones said, you can't just dissolve a sheet of metal or plasma in a cup.  If you want miners to have access to this, you'll have to figure out some other way for them to get reagents.

**Comment by lordpidey** [comment]
Iamgoofball, I agree, there should be mix messages for everything, but wouldn't that be better to do in another PR?

And I could have sworn there was a grinder on the mining station.  I guess I should have checked first.

**Comment by ojacobson** [comment]
@lordpidey solution: add grinder to the kitchen in the mining station! :D

**Comment by lordpidey** [comment]
I've listened to the suggestions on here.  
I've added a minor ingestion side effect to indicate that it's meant to be applied, just a short stun and weakness, no damage.

I've added a grinder to the mining station's kitchen.

I've changed the secondary recipe to be more "ghetto" chemistry.  It is now a sheet metal + a plasma sheet + a twinkie in a grinder.

**Comment by SconesC** [comment]
this is enabling a really horrific sleeper op miner strat that has nothing to do with the reagent in questions and i love it

**Comment by lordpidey** [comment]
What strategy would that be?

As far as I know the only chemicals that can be gotten in the mining outpost with a grinder are
Gold, silver, iron, plasma, sillicon, uranium, water, sugar, nutrament, space beer, and nicotine.

The only existing recipe that jumps out at me is EMP.
Without stabilizing agent, EMP runs will be quite difficult.

**Comment by Rinoka** [comment]
Traitors can get poison boxes and sleepy pens.

This lets you delude people into thinking nothing's wrong and they'll be fine until they go unconscious and die because you pricked them and ran. And I love it.

**Pull request merged**